docs/csharp/programming-guide/concepts/expression-trees/index.md

```diff
@@ -104,7 +104,7 @@ Console.WriteLine(factorial);
 // Prints 120.  
 ```  
   
- For more information, see [Generating Dynamic Methods with Expression Trees in Visual Studio 2010 (or later)](http://go.microsoft.com/fwlink/?LinkId=169513).  
+ For more information, see [Generating Dynamic Methods with Expression Trees in Visual Studio 2010](http://go.microsoft.com/fwlink/?LinkId=169513), which also applies to later versions of Visual Studio.
```

Comment:
I think the second would be the best to link to. 

And, yes, we should merge these in time.